ecs-init/config/common.go

```diff
@@ -104,6 +104,9 @@ const (
 	// defaultCredentialsFetcherSocketPath is set to /var/credentials-fetcher/socket/credentials_fetcher.sock
 	// in case path is not passed in the env variable
 	DefaultCredentialsFetcherSocketPath = "/var/credentials-fetcher/socket/credentials_fetcher.sock"
+
+    //SecurityOptsEnvVar is the environment variable that specifies a list of string values to customize labels for MLS systems, such as SELinux
+	SecurityOptsEnvVar = "ECS_AGENT_SELINUX_SECURITY_OPTS"
```

Comment:
Can we rename `ECS_AGENT_SELINUX_SECURITY_OPTS`  to `ECS_AGENT_DOCKER_SECURITY_OPTS` to be more generic since security opts might not be specific to SELINUX?

Reply:
Sure, will do